safe-global/web-core
# feat: error boundary

## What it solves

Inclusion of an Error Boundary `fallback`.

## How this PR fixes it

A `fallback` component has been created to match that of `safe-react` and is passed to `Sentry.ErrorBoundary`.

## How to test it

Locally edit the `AppProviders` to the following and observe the new component in the browser.

```ts
export const AppProviders = ({ children }: { children: ReactNode | ReactNode[] }) => {
  const theme = useLightDarkTheme()

  return (
    <ThemeProvider theme={theme}>
      <ErrorBoundary error={new Error('Test')} />
    </ThemeProvider>
  )
}
```

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/188405108-5a45c8f0-9391-4868-bf1d-d211b54b51e3.png)
